tools/tasks/seed/build.js.dev.ts

```diff
@@ -31,7 +31,7 @@ export = () => {
     .pipe(plugins.typescript(tsProject));
 
   return result.js
-    .pipe(plugins.sourcemaps.write())
+    .pipe(plugins.sourcemaps.write('.', {includeContent: false, sourceRoot: '../../src/client'}))
```

Comment:
does this cause any adverse effects? Like does `npm start` still run as normal from command line? just curious.